config/rbac/role.yaml

```diff
@@ -1,9 +1,71 @@
----
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: updateservice-operator
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - services
+  - secrets
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
```

Comment:
is this generated? it's redundant with one above which includes secrets

Reply:
I thought `get` should be enough for `secrets` (as we copy the globel pull secret from the `openshift-config` namespace) but https://github.com/openshift/cincinnati-operator/pull/191#discussion_r1665775787

Forgot to remove it after adding `secrets` into the above.